From c36c5072647d4b88c6e291b5c103b19e40cac4af Mon Sep 17 00:00:00 2001
From: Vicentiu Galanopulo <vicentiu@balena.io>
Date: Thu, 29 Apr 2021 15:53:16 +0200
Subject: [PATCH] Update device tree to use thermal cooling with PWM FAN

Add pwm-fan dts node and integrate it in the thermal-zones node.
Trip values are set for 65 and 70 deg. Celsius

Upstream-status: Inappropriate [configuration]
Signed-off-by: Vicentiu Galanopulo <vicentiu@balena.io>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi | 35 ++++++++++++++++++++++++++-
 1 file changed, 34 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index c7ec0f7..28c543b 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -426,7 +435,20 @@
 					hysteresis = <2000>;
 					type = "passive";
 				};
-				cpu_crit0: trip1 {
+
+				cpu_alert1: trip1 {
+					temperature = <60000>;
+					hysteresis = <1000>;
+					type = "passive";
+				};
+
+				cpu_alert2: trip2 {
+					temperature = <65000>;
+					hysteresis = <2000>;
+					type = "passive";
+				};
+
+				cpu_crit0: trip4 {
 					temperature = <95000>;
 					hysteresis = <2000>;
 					type = "critical";
-- 
2.7.4

